samples/excel/30-range/test-for-used-range.yaml

```diff
@@ -0,0 +1,125 @@
+id: excel-test-for-used-range
+name: Test for used range
+description: Create a chart from a table only if there's data in the table.
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#try-create-chart-from-empty-table").click(() => tryCatch(tryCreateChartFromEmptyTable));
+
+        async function tryCreateChartFromEmptyTable() {
+            await Excel.run(async (context) => {
+
+                const sheet = context.workbook.worksheets.getItem("Sample");    
```

Comment:
If you find the "const"-s with relation to variables odd (https://github.com/OfficeDev/office-js-snippets/issues/29), just replace them with "let".  I honestly think we over-pivoted on the "const" thing.  For my book, I've gone back and forth on this, but I use "let" most of the time because I think "const" is just confusing relative to what it means in other programming languages.

We can discuss real quick in person if you want.  Or just do what feels right to you for now, and we can then go back and take a look at what looks better (or send out a survey to MVPs and find out what they think)